### DISCUSSION TITLE
过期：ArchLinux 下遇到 [thumbnailer_worker] 报错

### DISCUSSION THREAD

**Discussion by txyyh** [hooke007/mpv_PlayKit#92] [回收]
### 背景
本人目前使用 ArchLinux 作为次主力系统，在将基于本项目的mpv配置文件迁移到 ArchLinux 上。

目前已通过配置文件修改后端接口 [1](https://github.com/txyyh/MPV_lazy/commit/fb593fb45bfcb70ca8980fc8b2d22e6e12e2f727) [2](https://github.com/txyyh/MPV_lazy/commit/b7884549b5d1cdf1de5384adddc95021cc8e407b)，手动安装本项目可用依赖的方式解决了绝大部分控制台 error 与 warning。

### 问题
目前在线播放时会遇到 [thumbnailer_worker] 报错，离线播放时会遇到 [thumbnailer_workerX] 报错（图如下所示）

看到脚本有写 Linux 判断逻辑，不过咱不懂这个所以也不清楚（（（

最后尝试安装的本项目可选依赖是 mimallow，但还未进行配置~~也不会配置~~

实在不知道问题出在哪儿，只好过来询问询问（

### 信息
<details><summary>控制台截图</summary>
<p>

![在线播放截图](https://user-images.githubusercontent.com/41893946/156798831-0b299816-b801-47d9-a150-91eab3e72b9c.png)
![离线播放截图](https://user-images.githubusercontent.com/41893946/156800824-920112e9-0ccd-4a99-8c04-b7080fcaff8f.png)
![离线播放图片](https://user-images.githubusercontent.com/41893946/156877281-72a3f438-ac34-4319-baf6-adfe90062434.png)

看着离线播放的报错每次都不太一样
</p>
</details>

<details><summary>脚本log信息</summary>
<p>

脚本log中[e]的大多是[mpv][ffmpeg]这些

``` logcat
134: 11: [   3.901][e][ffmpeg] tls: Error decoding the received TLS packet.
141: 11: [   3.901][e][mkv] Next subelement content goes past end of containing element, will be truncated
142: 11: [   3.901][e][mkv] Subelement content goes past end of containing element

245: 11: [   4.048][e][vo/lavc] error encoding at EOF
```
在线播放差不多这些，考虑缩略图没有正常加载，也正常

离线的没看到log

</p>
</details>
<details><summary>使用的 mpv 本体与安装的依赖</summary>
<p>

ArchLinux 已经是最新

archlinuxcn/mpv-git 0.34.0_220_gb15b3f6711-1

core/python 3.10.2-1 ~~是的vapoursynth没报版本错误~~

community/vapoursynth R57-2

extra/ffmpeg 2:5.0-4

coummunity/yt-dlp 2022.02.04-2

extra/tcl 8.6.12-3

extra/tk 8.6.12-1

aur/vapoursynth-plugin-mvtools 23-1

aur/vapoursynth-plugin-svpflow1 4.2.0.142-2

aur/vapoursynth-plugin-svpflow2-bin 4.3.0.168-1

aur/vapoursynth-plugin-temporalsoften2-git v1.1.ga133670-1

~~community/mimalloc 2.0.5-1~~ 对解决问题无用所以先卸了

</p>
</details>

**Comment by hooke007** [comment]
这个脚本报错时会在缩略图缓存目录生成脚本自己的log，你可以看看

**Reply by txyyh** [reply]
脚本log中[e]的大多是[mpv][ffmpeg]这些

``` logcat
134: 11: [   3.901][e][ffmpeg] tls: Error decoding the received TLS packet.
141: 11: [   3.901][e][mkv] Next subelement content goes past end of containing element, will be truncated
142: 11: [   3.901][e][mkv] Subelement content goes past end of containing element

245: 11: [   4.048][e][vo/lavc] error encoding at EOF
```
在线播放差不多这些，考虑缩略图没有正常加载，也正常

离线的没看到log

**Comment by nyfair** [comment]
编译的时候用luajit，别用lua

**Reply by txyyh** [reply]
无效，报错依旧

**Comment by zhongfly** [comment]
我记得这个缩略图脚本本来就不适配在线播放吧？

**Reply by txyyh** [reply]
离线也有报错
在线确实（（ 要不我删了那段？

**Reply by nyfair** [reply]
只要网速快，支持断点续传就能用，自家NAS一试便知
